firebase_insert_block.py

```diff
@@ -7,7 +7,7 @@
 @discoverable
```

Comment:
Actually do not need `@discoverable` on this, the behavior currently is that block discovery isn't related to inheritance at all, and the default is to be discoverable. So you only need to specify non-discoverable blocks.

Reply:
Does this remove the need for `from nio.util.discovery import discoverable` in these files as well?

Reply:
Indeed!